httpserver/src/test/kotlin/tbdex/sdk/httpserver/handlers/CreateExchangeTest.kt

```diff
@@ -12,11 +12,12 @@ import io.ktor.http.contentType
 import kotlinx.coroutines.runBlocking
 import org.junit.jupiter.api.Test
 import tbdex.sdk.httpclient.models.ErrorResponse
+import tbdex.sdk.httpserver.models.CreateExchangeRequest
 import tbdex.sdk.protocol.serialization.Json
 import kotlin.test.assertContains
 import kotlin.test.assertEquals
 
-class SubmitRfqTest : ServerTest() {
+class CreateExchangeTest : ServerTest() {
   @Test
   fun `returns BadRequest if no request body is provided`() = runBlocking {
     val response = client.post("/exchanges/123/rfq") {
```

Comment:
Update endpoints in this file as per spec
https://github.com/TBD54566975/tbdex/pull/222/files#diff-1ab7678648ffe33c3535373210080ab41cd2c38573890fdfcf008700ef14c3c7R279

Reply:
actually i should just get rid of the `/rfq` bit since we've decided that can go, but the one up the air is whether `/{exchangeId}` should go also. updating

Reply:
updated